numpy_questions.py

```diff
@@ -64,4 +71,12 @@ def wallis_product(n_terms):
     """
     # XXX : The n_terms is an int that corresponds to the number of
     # terms in the product. For example 10000.
-    return 0.
+    pi = 1
+
+    if n_terms == 0:
```

Comment:
@mathieu-peh can you explain this line ?

Reply:
Which line ? The "return 0" one ? I don't have this line in my original code

Reply:
The github comment appears exactly under the line it refers to, so line 76 here